samples/jvm/spek/build.gradle

```
@@ -0,0 +1,55 @@
+buildscript {
+    ext {
+        kotlin_version = '1.3.41'
+        spek_version = '2.1.0-alpha.0.10+cc4a596'
+        atrium_version='0.9.0-alpha'
+    }
+
+    repositories {
+        jcenter()
+    }
+
+    dependencies {
+        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
+    }
+}
+
+plugins {
+    id 'org.jetbrains.kotlin.jvm' version '1.3.41'
+}
+
+group 'com.atrium.spek'
+version '1.0-SNAPSHOT'
+
+apply plugin: 'kotlin'
+
+repositories {
+    // atrium jvm installation: more info here -> https://www.kotlinresources.com/library/atrium/#jvm
+    jcenter()
+
+    // dependencies for spek
+    mavenLocal()
+    // uncomment to use development build for <u>spek<u>
+    maven { url "https://dl.bintray.com/spekframework/spek-dev" }
```

either change the comment or comment this out :)
```suggestion
    // maven { url "https://dl.bintray.com/spekframework/spek-dev" }
```